website/content/docs/nia/api/status.mdx

```diff
@@ -21,18 +21,24 @@ This endpoint currently returns the overall status information for all tasks.
 #### Request Parameters
 Currently no request parameters are offered for the overall status API.
 
-#### Response Fields
+#### Response Statuses
+
+| Status | Reason           |
+| ------ | ---------------- |
+| 200    | Successfully retrieved the status |
 
-* `task_summary` - Summary of task information. See [Task Status API](/docs/nia/api#task-status) to learn more about how health status is determined.
-  * `status` - Summary count of how many tasks have each health status value
-    * `successful` - (int) The number of tasks that have a 'successful' health status
-    * `errored` - (int) The number of tasks that have a 'errored' health status
-    * `critical` - (int) The number of tasks that have a 'critical' health status
-    * `unknown` - (int) The number of tasks that have an 'unknown' health status
-  * `enabled` - Summary count of how many tasks are enabled or disabled
-    * `true` - (int) The number of tasks that are enabled
-    * `false` - (int) The number of tasks that are disabled
+#### Response Fields
 
+| Name	| Type	 | Description        |
+| ----- | ------ | ------------------ |
+|`status` | object |  Summary count of how many tasks have each health status value.
```

Comment:
```suggestion
|`status` | object |  The summary count of how many tasks that each have a health status value.
```
Cleaner?

Reply:
I think this implies that not all tasks have a health status, which isn't the case. Maybe something like `The summary of how many tasks have a given health status value`?